python/llm/example/transformers/general_int4/README.md

```diff
@@ -0,0 +1,20 @@
+# BigDL-LLM General INT4 Optimization for Large Language Model
```

Comment:
move to `llm/example/pytorch`

Reply:
`General INT4` is confusing

Reply:
> move to `llm/example/pytorch`

Have moved and removed all `General INT4` descriptions.